package.json

```diff
@@ -263,6 +263,10 @@
                         "vs-kubernetes.resources-to-watch": {
                             "type": "array",
                             "description": "List of resources to be watched."
+                        },
+                        "vs-kubernetes.enable-snap-flag": {
+                            "type": "boolean",
+                            "description": "If true will enable the flag to run kubectl commands using kubectl installed by snap."
```

Comment:
```suggestion
                            "description": "Enables compatibility with instances of VS Code that were installed using snap."
```

Reply:
Might be worth adding something to the read-me, or starting a new Troubleshooting or FAQ page that we can link from the read-me.